packages/block-library/src/list-item/utils.js

```diff
@@ -1,40 +1,27 @@
 /**
  * WordPress dependencies
  */
-import { createBlock, switchToBlockType } from '@wordpress/blocks';
-
-/**
- * Internal dependencies
- */
-import { name as listItemName } from './block.json';
-import { name as listName } from '../list/block.json';
-import { name as paragraphName } from '../paragraph/block.json';
-
-export function createListItem( listItemAttributes, listAttributes, children ) {
-	return createBlock(
-		listItemName,
-		listItemAttributes,
-		! children?.length
-			? []
-			: [ createBlock( listName, listAttributes, children ) ]
-	);
-}
```

Comment:
This `createListItem` function is unused. Nobody ever imports it from the `utils.js` module, and it's not a part of public API either.